### PR TITLE
Smart Sense U-Boot Port

### PR DESCRIPTION
**Pull request by MikeLoder**
- Adding in support for Smart Sense in the u-boot configuration
- Porting commits from the Smart Sense repo to this u-boot fork.

### CODE REVIEW

File: board/miovision/smartsense/MAINTAINERS [hunk 1]
```diff
@@ -0,0 +1,6 @@
+SMART SENES BOARD
```
Comment (m3harvey) [inline board/miovision/smartsense/MAINTAINERS]:
```suggestion
SMART SENSE BOARD
```

File: board/miovision/smartsense/Kconfig [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2020, Miovision.
+# Copyright (c) 2020, Miovision
```
Comment (m3harvey) [inline board/miovision/smartsense/Kconfig:2]: Did you want to have the copyright notice twice?

**Review by m3harvey — APPROVED** [review]

**Review by dburrell-mio — APPROVED** [review]

File: board/miovision/smartsense/Kconfig (hunk 2 above)
Comment (MikeLoder) [reply board/miovision/smartsense/Kconfig:2]: Fixed

File: arch/arm/dts/tegra186-smartsense.dts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+/dts-v1/;
+
+#include "tegra186-p2771-0000.dtsi"
+
+/ {
+	model = "MIOVISION SMART SENSE";
```
Comment (smacdonald-miov) [inline arch/arm/dts/tegra186-smartsense.dts]:
```suggestion
	model = "MIOVISION SMARTSENSE";
```

File: arch/arm/mach-tegra/tegra186/Kconfig [hunk 4]
```diff
@@ -15,6 +15,11 @@ config TARGET_P2771_0000
 	  micro-B port, Ethernet, USB3 host port, SATA, PCIe, and two GPIO
 	  expansion headers.
 
+config TARGET_SMARTSENSE
+	bool "Miovision Tegra186 Smart Sense board"
```
Comment (smacdonald-miov) [inline arch/arm/mach-tegra/tegra186/Kconfig]:
```suggestion
	bool "Miovision Tegra186 SmartSense board"
```

File: arch/arm/mach-tegra/tegra186/Kconfig [hunk 5]
```diff
@@ -15,6 +15,11 @@ config TARGET_P2771_0000
...
+	help
+	  Miovision Smart Sense Board
```
Comment (smacdonald-miov) [inline arch/arm/mach-tegra/tegra186/Kconfig]:
```suggestion
	  Miovision SmartSense Board
```

File: board/miovision/smartsense/MAINTAINERS [hunk 6]
```diff
@@ -0,0 +1,6 @@
+SMART SENSE BOARD
```
Comment (smacdonald-miov) [inline board/miovision/smartsense/MAINTAINERS]:
```suggestion
SMARTSENSE BOARD
```

**Review by m3harvey — APPROVED** [review]

**Review by smacdonald-miov — APPROVED** [review]

**Pull request merged**